lib/extensions/descendant_loader.rb

```diff
@@ -71,7 +71,12 @@ def classes_in(filename)
       require 'ripper_ruby_parser'
 
       content = File.read(filename)
-      parsed = RipperRubyParser::Parser.new.parse(content)
+      begin
+        parsed = RipperRubyParser::Parser.new.parse(content)
+      rescue => e
+        puts "\nError parsing classes in #{filename}:\n#{e.class.name}: #{e}\n\n"
```

Comment:
Should it be `STDERR.puts`?

Reply:
done (used `$stderr`, it's my impression that's more polite because easily re-assignable)